src/js/setup-text-tracks.js

```diff
@@ -28,7 +28,7 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
     .map((track) => ({
       dashTrack: track,
       trackConfig: {
-        label: track.lang,
+        label: track.label,
```

Comment:
This should use the language if the `label` isn't populated, i.e.
```suggestion
        label: track.label || track.lang,
```

Reply:
Thanks @squarebracket , you're right.  Updated.